client/basic-authorship/src/basic_authorship.rs

```diff
@@ -95,12 +106,29 @@ impl<A, B, C> ProposerFactory<A, B, C, DisableProofRecording> {
 			transaction_pool,
 			metrics: PrometheusMetrics::new(prometheus),
 			default_block_size_limit: DEFAULT_BLOCK_SIZE_LIMIT,
+			soft_deadline_percent: DEFAULT_SOFT_DEADLINE_PERCENT,
 			telemetry,
 			client,
 			include_proof_in_block_size_estimation: false,
 			_phantom: PhantomData,
 		}
 	}
+
+	/// Set soft deadline percentage.
```

Comment:
This needs to be moved into the impl block below in line 177.

Reply:
Thanks! Done.